src/TrackPanel.cpp

```diff
@@ -3751,7 +3751,7 @@ void TrackPanel::ForwardEventToWaveTrackEnvelope(wxMouseEvent & event)
    //  ie one of the Wave displays
    if (display <= 1) {
       bool dB = (display == 1);
-      bool needUpdate = false;
```

Comment:
This was probably solving a "possible uninitialized variable" compiler warning since it's only set within if statements.